ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -370,6 +370,44 @@ exports[`WorkflowProjectGenerator > function > should generate <function_name>.p
 "
 `;
 
+exports[`WorkflowProjectGenerator > function > should generate deployment workflow tool > code/nodes/workflow/__init__.py 1`] = `
+"from typing import List
+
+from vellum import ChatMessage, ChatMessagePromptBlock, PlainTextPromptBlock, RichTextPromptBlock, VariablePromptBlock
+from vellum.workflows.nodes.experimental.tool_calling_node.node import ToolCallingNode
+from vellum.workflows.types.definition import DeploymentDefinition
+
+
+class MyToolCallingNode(ToolCallingNode):
+    ml_model = "gpt-4"
+    blocks = [
```

Comment:
[nit] in general, this test is focusing on the deployment tool. This means we could significantly reduce the size of the `blocks` (is empty possible?) and `prompt_inputs` attributes